openpype/pipeline/colorspace.py

```diff
@@ -568,46 +617,61 @@ def get_labeled_colorspaces(
                 if color_data.get("aliases"):
                     aliases.update([
                         (
-                            alias_name,
+                            "aliases::{}".format(alias_name),
                             "[alias] {} ({})".format(alias_name, color_name)
                         )
                         for alias_name in color_data["aliases"]
                     ])
-                colorspaces.add(color_name)
+                colorspaces.add((
+                    "{}::{}".format(items_type, color_name),
+                    "[colorspace] {}".format(color_name)
+                ))
 
         elif items_type == "looks":
             looks.update([
-                (name, "[look] {} ({})".format(name, role_data["process_space"]))
+                (
+                    "{}::{}".format(items_type, name),
+                    "[look] {} ({})".format(name, role_data["process_space"])
+                )
                 for name, role_data in colorspace_items.items()
             ])
 
         elif items_type == "displays_views":
             display_views.update([
-                (name, "[view (display)] {}".format(name))
+                (
+                    "{}::{}".format(items_type, name),
+                    "[view (display)] {}".format(name)
+                )
                 for name, _ in colorspace_items.items()
             ])
 
         elif items_type == "roles":
             roles.update([
-                (name, "[role] {} ({})".format(name, role_data["colorspace"]))
+                (
+                    "{}::{}".format(items_type, name),
+                    "[role] {} ({})".format(name, role_data["colorspace"])
+                )
                 for name, role_data in colorspace_items.items()
             ])
 
     if roles and include_roles:
+        roles = sorted(roles, key=lambda x: x[0])
         labeled_colorspaces.extend(roles)
 
-    # add colorspace after roles so it is first in menu
-    labeled_colorspaces.extend((
-        (name, f"[colorspace] {name}") for name in colorspaces
-    ))
+    # add colorspaces as second so it is not first in menu
+    colorspaces = sorted(colorspaces, key=lambda x: x[0])
```

Comment:
Since the values are tuples can't we just sort on both items. It'd only check the second item if the first one matched anyway? Which means we can just do `sorted(colorspaces)` etc. Right?

Reply:
Also note that the sorting is Case sensitive - which I'm not sure whether a user would want that visually?